### PR TITLE
feat: fake news

### PR DESCRIPTION
**Pull request by sanghin**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 const Discord = require('discord.js');
 const redis = require('redis');
 const SHA256 = require('crypto-js/sha256');
 const moment = require('moment-timezone');
 
 const PATH_TO_STATIC_FOLDER = `${__dirname}/static/img/`;
-const { DISCORD_TOKEN = null, REDIS_URL = 'redis://redis:6379' } = process.env;
+const { DISCORD_TOKEN, REDIS_URL } = process.env;
 const client = new Discord.Client();
-const token = DISCORD_TOKEN;
```
Comment (ltribolet) [inline server.js]: 👍

File: server.js [hunk 2]
```diff
@@ -68,7 +68,15 @@ client.on('message', (message) => {
   }
 
   if (message.content.replace(/\s/g, '').match(IS_NESTCEPAS_REGEX)) {
-    message.channel.send('', { file: `${PATH_TO_STATIC_FOLDER}/nestcepas.gif` });
+    message.channel.send('', {
```
Comment (ltribolet) [inline server.js]: Not very fond of the code style here, semi inline and line return. Ssomething more coherent would be appreciated.

File: server.js [hunk 3]
```diff
@@ -68,7 +68,15 @@ client.on('message', (message) => {
...
+      file: `${PATH_TO_STATIC_FOLDER}/nestcepas.gif`,
+    });
+  }
+
+  if (message.content.match(IS_FAKE_NEWS_REGEX)) {
+    // will select randomly between fakenews_1.jpg and fakenews_2.jpg
+    const image = `fakenews_${Math.floor(Math.random() * 2) + 1}`;
```
Comment (ltribolet) [inline server.js]:
```
let image = ['fakenews_1','fakenews_2'][Math.round(Math.random())];
```

File: server.js (hunk 2 above)
Comment (sanghin) [reply server.js]: Dunno why this line was split

File: server.js (hunk 3 above)
Comment (sanghin) [reply server.js]:
```
const image = ['fakenews_1', 'fakenews_2'][Math.round(Math.random())];
```

**Pull request merged**